extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -314,6 +316,22 @@ describe('amp-a4a', () => {
       });
     });
 
+    it('populate postAdResponseExperimentFeatures', () => {
+      adResponse.headers[EXPERIMENT_FEATURE_HEADER_NAME] =
+          `foo=bar,bad,${CSP_ENABLED_EXP_NAME}=true`;
+      a4a.buildCallback();
+      a4a.onLayoutMeasure();
+      return a4a.layoutCallback().then(() => {
+        const child = a4aElement.querySelector('iframe[srcdoc]');
+        expect(child).to.be.ok;
+        expect(child.srcdoc.indexOf('meta http-equiv=Content-Security-Policy'))
```

Comment:
I would do a querySelector here instead so that it's not sensitive to how you choose to quote stuff in the HTML.

Reply:
done

Reply:
For some reason its giving me issues when I try child.contentDocument.querySelector as if its selecting a different frame.  I tested locally and things work fine so I'm going to leave like this.